exauq/core/emulators.py

```diff
@@ -299,6 +300,12 @@ def fit(
             self._gp.theta
         )
 
+        if len(training_data) < (len(training_data[0].input) + 3):
+            warn(
+                f"Warning: number of parameters being estimated is larger than the size of the training data ({inputs.shape[0]}), "
+                f"estimates should not be trusted."
+            )
```

Comment:
Couple of things... It would be good to provide the "number of parameters being estimated". And prefixing with `Warning:` is redundant when using `warnings.warn()` as it already tags it as a warning. 
```suggestion
            warn(
               f"Fewer training points ({len(training_data)}) than hyperparameters being estimated ({n_params_estimated}). "
               "Estimates may be unreliable."
            )
```

Reply:
Fixed.